### ISSUE TITLE
Translation actorconditions and monsterlist

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#294]
```
Translation content_actorconditions.xml and content_monsterlist.xml (r209) to 
russian

```

Original issue reported on code.google.com by `shell.an...@gmail.com` on 2 Jan 2012 at 2:32

Attachments:
- [content_actorconditions.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-294/comment-0/content_actorconditions.xml)
- [content_monsterlist.xml](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-294/comment-0/content_monsterlist.xml)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r213.
```

Original comment by `oskar.wi...@gmail.com` on 6 Jan 2012 at 8:48
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.11**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
